appservice/src/extensionVariables.ts

```diff
@@ -47,4 +48,5 @@ export function registerAppServiceExtensionVariables(extVars: IAppServiceExtensi
 
     ext = extVars;
     registerUIExtensionVariables(extVars);
+    registerAzureUtilsExtensionVariables(extVars);
```

Comment:
I think this is the right way to handle the split, right?

Reply:
That makes sense. Since they all no-op if they've already been called, it should be safe.